__tests__/app.test.js

```diff
@@ -58,4 +58,19 @@ describe("NC_Games API", () => {
         });
     });
   });
+  describe("GET /api/users", () => {
+    it("200: Returns all category objects with username, name and avatar_url", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.users.length > 0).toBe(true);
```

Comment:
Have a look at the test data and look at how many users there are. I'd check you're getting them all. 
At the moment, you could only be getting one with this test